### ISSUE TITLE
BUG: Say something here

### ISSUE THREAD

**Issue by RumosZin** [beekeeper-studio/beekeeper-studio#2435]
**Describe the bug**
Just as we can specify asc, desc when creating multiple column indexes from mysql console, I wish we could specify asc in the UI interface tool.

**To Reproduce**
1. Go to Table's Indexes menu
2. Create multiple index with asc, desc

![image](https://github.com/user-attachments/assets/973b3ffd-f20c-4c71-b245-c68990b09f3d)

**Expected behavior**
This multiple column index is normally registered as an index, and ASC is omitted from the UI interface as before.

**Version Information (please complete the following information):**
 - OS: Mac
 - App Version : Beekeeper Studio Ultimate 4.6.4
 - Database type and version : mysql from 11.4.2-MariaDB, client 15.2 for osx10.19 (arm64) using  EditLine wrapper

**Additional context**
Can I do it if I can?

**Comment by not-night-but** [comment]
@RumosZin Yeah I think this area of the structure view could use some work for sure.
